systemtests/src/test/java/io/enmasse/systemtest/isolated/upgrade/UpgradeTest.java

```diff
@@ -505,11 +509,31 @@ private void upgradeEnmasseOLM(Path upgradeTemplates, String previousVersion) th
         //update subscription to point to new catalog and to use latest csv
         operatorManager.olm().applySubscription(infraNamespace, catalogNamespace, csvName, Environment.getInstance().getOperatorName(), Environment.getInstance().getOperatorChannel());
 
+
+        if(Environment.getInstance().isTestDownstream().equals("true")){
+            operatorManager.olm().applyDownstreamSubscription(infraNamespace);
+        }
+        else {
+            operatorManager.olm().applySubscription(infraNamespace, catalogNamespace, csvName, Environment.getInstance().getOperatorName(), Environment.getInstance().getOperatorChannel());
+        }
+
         //should update
         Thread.sleep(300_000);
+
+        if(Environment.getInstance().isTestDownstream().equals("true")){
+            String installPlanName = "amq-online";
+            approveInstallPlan(infraNamespace, installPlanName);
+        }
+
         checkImagesUpdated(getVersionFromTemplateDir(upgradeTemplates));
     }
 
+    private void approveInstallPlan(String namespace, String installPlanName) {
+        log.info("Approving install plan {} in namespace {}", installPlanName, namespace);
+        List<String> cmd = Arrays.asList("/bin/bash", "-c", " patch installplan " + installPlanName + " --namespace " + namespace + " --type json  --patch '[{\"op\": \"add\", \"path\": \"/spec/approved\", \"value\": true}]'");
```

Comment:
you should use our kubecmd client because this command is nonsence, you need to call oc patch a not bin/bash -c path